copy/copy.go

```diff
@@ -182,6 +183,8 @@ type Options struct {
 	// OciDecryptConfig contains the config that can be used to decrypt an image if it is
 	// encrypted if non-nil. If nil, it does not attempt to decrypt an image.
 	OciDecryptConfig *encconfig.DecryptConfig
+
+	FetchPartialBlobs bool // attempt to fetch the blob partially.  Experimental.
```

Comment:
I have no idea what this means, and the test Skopeo PR hard-codes this to true. Is this intended to be an end-user option, or just a feature flag for a WIP feature?

Reply:
It is hard coded just for testing purposes.  It should be an option, at least for now